portfolio/src/main/webapp/script.js

```diff
@@ -41,6 +41,22 @@ setInterval(autoChange, 5000);
 
 function getName() {
   fetch("/data").then(response => response.text()).then((name) => {
-    document.getElementById('NAME').innerText = name;
+    document.getElementById('name').innerText = name;
   });
+}
+
+function makeList(){
+    fetch("/data").then(response => response.json()).then((wordList) => {
+        const buildList = document.getElementById("make-me");
+        buildList.innerHTML = "";
+        buildList.appendChild(createli(wordList[0]));
+        buildList.appendChild(createli(wordList[1]));
+        buildList.appendChild(createli(wordList[2]));
```

Comment:
Why not just loop through the wordList array here? Seems like it'd be a bit of future proofing if you add or remove words later. It also de-couples your front end implementation a bit from your server side (often useful in real projects where the backend may be owned by a different team).